src/main/java/appeng/container/implementations/WirelessCraftConfirmContainer.java

```diff
@@ -201,18 +200,12 @@ public void startJob() {
         ScreenHandlerType<?> originalGui = null;
 
         final IActionHost ah = getActionHost();
-
-        if(ah instanceof WCTGuiObject) {
-            originalGui = WCTContainer.TYPE;
-        } else if(ah instanceof WPTGuiObject) {
+        if (ah instanceof WCTGuiObject || ah instanceof WPTGuiObject)
             originalGui = WPTContainer.TYPE;
-        }
```

Comment:
this won't work, since it there is a difference between WCTContainer and WPTContainer

Reply:
m ... I don't think you looked closely

Reply:
but stop, I got it mixed up, sorry